test/Garnet.test/RespVectorSetTests.cs

```diff
@@ -1481,6 +1481,64 @@ public void RecreateIndexesOnRestore()
 
         // TODO: FLUSHDB needs to cleanup too...
 
+        [Test]
+        public void VINFO()
+        {
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var db = redis.GetDatabase();
+
+            // Create a vector set with known parameters
```

Comment:
I suggest trying all combinations of quantizers, reduce, effort (default and explicit values), and num links (default and explicit values) and making sure they're correct in VINFO.